README.md

```diff
@@ -35,6 +37,13 @@ $ helm plugin install https://github.com/databus23/helm-diff --version master
 Pick the deployment tool that suits your environment and then change directory and follow the Readme.md files there.
 
 
+Namespaces:
```

Comment:
We create a 'besu' namespace now and deploy in there rather than 'default'. monitoring as is in its own namespace.